jakartaee/10/concurrency/22.0.0.3-beta-Java11-TCKResults.adoc

```diff
@@ -0,0 +1,287 @@
+:page-layout: certification 
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of Jakarta EE Concurrency.
+
+== Open Liberty 22.0.0.3 Beta - Jakarta EE Concurrency Certification Summary (Java 11)
+
+* Product Name, Version and download URL (if applicable):
++
+https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/beta/2022-02-01_1901/openliberty-22.0.0.3-beta.zip[Open Liberty 22.0.0.3 Beta]
+
+* Specification Name, Version and download URL:
++
+link:https://jakarta.ee/specifications/Concurrency/3.0[Jakarta EE Concurrency 3.0]
```

Comment:
Odd to have `Concurrency` capitalized

Reply:
It's the specification name, so it gets capitalized here

Reply:
I meant in the URL

Reply:
it should be lower case in the URL